pkg/costmodel/router.go

```diff
@@ -1156,6 +1156,66 @@ func (a *Accesses) GetInstallNamespace(w http.ResponseWriter, r *http.Request, _
 	w.Write([]byte(ns))
 }
 
+type InstallInfo struct {
+	Containers  []ContainerInfo   `json:"containers"`
+	ClusterInfo map[string]string `json:"clusterInfo"`
+}
+
+type ContainerInfo struct {
+	ContainerName string `json:"containerName"`
+	Image         string `json:"image"`
```

Comment:
What do you think about including the `imageID` from [ContainerStatus](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#containerstatus-v1-core)? I think it could be helpful in case we're looking at an image tagged `latest` and want to find the exact image.

Example live image and imageID in my cluster:
```
image: gcr.io/kubecost1/frontend:prod-1.94.3
imageID: gcr.io/kubecost1/frontend@sha256:2b39858fb8599f6789a3a565ab6648b7d5a3c1b333faa95ec15d5c8a510277da
```

Reply:
When looking at a pods status it will not say `latest` for the image, it will say the actual image including the tag tag that is being ran. ImageID is really useful when people push over the same image which we don't but I can add it anyways.